reana_client/cli/workflow.py

```diff
@@ -914,8 +914,19 @@ def workflow_logs(
     help="If set, try to pull remote environment image from registry to perform "
     "validation locally. Requires ``--environments`` flag. [default=False]",
 )
+@click.option(
```

Comment:
IMHO this flag is not needed as it seems that workspace path validation is a matter of general reana.yaml validation and it's not that expensive to be excluded as `environments`.